presto-product-tests/src/assembly/scripts.xml

```diff
@@ -0,0 +1,28 @@
+<assembly>
```

Comment:
In presto-docs/src/main/assembly/sources.xml, this assembly element has more tags:
`<assembly xmlns="http://maven.apache.org/plugins/maven-assembly-plugin/assembly/1.1.2"
          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
          xsi:schemaLocation="http://maven.apache.org/plugins/maven-assembly-plugin/assembly/1.1.2 http://maven.apache.org/xsd/assembly-1.1.2.xsd">`